moulinette/interfaces/cli.py

```diff
@@ -335,6 +335,7 @@ def run(self, args, output_as=None, password=None):
                 - plain: return a script-readable output
                 - none: do not output the result
             - password -- The password to use in case of authentication
+            - timeout -- Timeout this command after 30 seconds if the lock isn't acquired
```

Comment:
Ah no, it's the value passed to the argument, forgot to update the doc

It would be

    yunohost domain list --timeout 42